src/test/java/io/stargate/sgv2/jsonapi/api/v1/FindIntegrationTest.java

```diff
@@ -772,5 +780,139 @@ public void findByBooleanColumn() {
           .statusCode(200)
           .body("data.docs[0]", jsonEquals(expected));
     }
+
+    @Test
+    @Order(3)
+    public void sort() {
+      deleteAllDocuments();
+      Map<String, String> sorted = getDocuments(25, true);
+      insert(sorted);
+      String json =
+          """
+            {
+              "find": {
+                "sort" : ["username"]
+              }
+            }
+          """;
+      JsonNodeFactory nodefactory = objectMapper.getNodeFactory();
+      final ArrayNode arrayNode = nodefactory.arrayNode(20);
+      final Iterator<Map.Entry<String, String>> iterator = sorted.entrySet().iterator();
+      try {
+        for (int i = 0; i < 20; i++)
+          arrayNode.add(objectMapper.readTree(iterator.next().getValue()));
+      } catch (Exception e) {
+        // ignore the object node creation error should never happen
+      }
+      given()
+          .header(HttpConstants.AUTHENTICATION_TOKEN_HEADER_NAME, getAuthToken())
+          .contentType(ContentType.JSON)
+          .body(json)
+          .when()
+          .post(CollectionResource.BASE_PATH, keyspaceId.asInternal(), collectionName)
+          .then()
+          .statusCode(200)
+          .body("data.count", is(20))
+          .body("data.docs", jsonEquals(arrayNode.toString()));
```

Comment:
Is there better way to check if the documents present in sorted order?

Reply:
I can't think of anything obvious. I guess it is possible to build all kinds of helper methods to filter out other fields and so on but I think Rest-assured mostly focuses on equality comparisons and not ordering.